Autopilot/PathManager/Src/landingTakeoffManager.cpp

```diff
@@ -120,6 +120,15 @@ _LandingPath LandingTakeoffManager::createSlopeWaypoints(const Telemetry_PIGO_t
 _LandingTakeoffOutput LandingTakeoffManager::translateWaypointCommands(const _WaypointManager_Data_Out & outputData)
 {
     _LandingTakeoffOutput controlOutput;
```

Comment:
You can use value initialization to prevent writing all of these lines of code. You do this by adding `{}`:

```cpp
_LandingTakeoffOutput controlOutput {};
```

Value initialization sets all members to their default value (0 for ints and false for bools)

Reply:
okok thanks!